src/cat.ts

```diff
@@ -17,35 +16,41 @@ export interface CatInput {
   theta?: number;
   minTheta?: number;
   maxTheta?: number;
-  prior?: number[][];
+  priorDist?: string;
+  priorPar?: number[];
   randomSeed?: string | null;
 }
 
 export class Cat {
+  [x: string]: any;
```

Comment:
I think we should remove this. `[x:string]: any;` is an index signature that allows for arbitrary properties to be added to instances of the class. This is generally not considered good practice in TypeScript because it undermines the type safety that TypeScript provides.
```suggestion
```